src/bosh-director/spec/unit/config_server/variables_interpolator_spec.rb

```diff
@@ -6,6 +6,7 @@
   let(:client_factory) { double(Bosh::Director::ConfigServer::ClientFactory) }
   let(:config_server_client) { instance_double(Bosh::Director::ConfigServer::ConfigServerClient) }
   let(:deployment_name) {'my_deployment_name'}
+  let(:instance_name) {'my_instance_name'}
```

Comment:
I would expect to see a new test case here which verifies the use of the cache?

Reply:
So we wrestled with this cause a lot of the calls are just mocked, This could be added if you think we would get a lot of value out of this

Reply:
If you looked into this and didn't see much value in adding the tests is fine with me. I just wanted to make sure that we don't forget them.